bench/run.js

```diff
@@ -2,6 +2,7 @@
 const childProcess = require('child_process');
 const path = require('path');
 const fs = require('fs');
+const Promise = require('bluebird');
```

Comment:
Don't do unrelated changes.

Reply:
Shall I remove them and add them in another PR?